app/test/frontend/golden/analysis_tab_http.html

```diff
@@ -78,49 +78,83 @@ <h4>Platforms</h4>
 </blockquote>
 
 <h4>Suggestions</h4>
-<ul>
-    <li>
-      <div><p>Running <code>dartdoc</code> failed.</p>
-</div>
-      <blockquote><p>Make sure <code>dartdoc</code> runs without any issues.</p>
-</blockquote>
-    </li>
-    <li>
-      <div><p>Fix analysis and formatting issues.</p>
+<div class="suggestion-row">
+  <div class="suggestion-title">
+    <span class="suggestion-icon-danger"></span>
+    <p>Running <code>dartdoc</code> failed.</p>
+
```

Comment:
Can we figure out why these needless blank lines are being inserted?

Reply:
It is a combination of:
- the output of `markdown`,
- the output of `mustache`,
- keeping our templates sane.

Unfortunately we have a lot of these, I'll revisit that later (https://github.com/dart-lang/pub-dartlang-dart/issues/1418)